### ISSUE TITLE
Actual need vs. wanted quantity

### ISSUE THREAD

**Issue by fabian** [worldskills/worldskills-infrastructure-list#88]
Sometimes the quantity in IL is higher than the actual need, because a sponsor is willing to provide more items to showcase their product.

This can lead to necessary high expectations for the next Competition Organizer, as the quantity in IL is higher than actually needed.

Maybe introduce another field "Actual quantity needed"?

**Comment by ailtisWSFR** [comment]
For me, the number in the field “Quantity” should be the actual quantity needed.  
If the sponsor gives more than that, couldn’t we just add a new field like the “Additional quantity” one? Something like “Additional quantity proposed by the sponsor” (and rename “Additional quantity” by “Additional quantity needed”)?

**Comment by fabian** [comment]
Yes, I would probably agree on that. I'm wondering if we even need to differentiate between “Additional quantity proposed by the sponsor” and “Additional quantity needed”?
